docs(EE): Update multitenancy with query-limit notes (#144)
dgraph-io/dgraph-docs: content/enterprise-features/multitenancy.md

special access to create or delete namespaces and change passwords of
users of other namespaces.

{{% notice "note" %}}
Dgraph provides a timeout limit per query that's configurable using the `--limit` superflag's `query-limit` option.
There's no time limit for queries by default, but you can override it when running Dgraph Alpha.

> For multi-tenant environments the recommended `query-limit` value is 500ms. 

Author: ```suggestion
For multi-tenant environments a suggested `query-limit` value is 500ms. 
```
